ftw/linkchecker/command/checking_links.py

```diff
@@ -0,0 +1,35 @@
+from zope.component.hooks import setSite
+from Testing.makerequest import makerequest
+from AccessControl.SecurityManagement import newSecurityManager
+from zope.globalrequest import setRequest
+import AccessControl
+from zc.relation.interfaces import ICatalog
+from zope.component import getUtility
+
+
+def main(plone, *args):
+
+    # Set up request for debug / bin/instance run mode.
+    app = makerequest(plone)
+    setRequest(app.REQUEST)
+
+    houptding = app.restrictedTraverse('/')
+
+    # here all pages need to be found
+    # for now there is only 'Plone'
+    # later on we need to iterate through
+    # each page and send reports to different people
+
+    MINI_SITE = houptding.get('Plone')
```

Comment:
Nope. Use sensible english identifiers. Like `portal = ` (uppercase is used for constants by convention). See https://devdocs.4teamwork.ch/programmierrichtlinien/#einhalten-der-pep8-konventionen

Reply:
👍